components/PostCardList/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./PostCardListItem";
```

Comment:
I think we need to export `PostCardList` as well. If anything, `PostCardList` should be default export, and `PostCardListItem` is supposed to be named-export.

```suggestion
export { default } from "./PostCardList";
export { default as PostCardListItem } from "./PostCardListItem";
```